sdk/tests/deqp/functional/gles3/es3fNegativeTextureApiTests.js

```diff
@@ -1150,6 +1150,11 @@ goog.scope(function() {
             gl.texImage2D(gl.TEXTURE_2D, 0, gl.RGBA32F, 0, 0, 0, gl.RGBA, gl.FLOAT, null);
             gl.generateMipmap(gl.TEXTURE_2D);
             this.expectError(gl.INVALID_OPERATION);
+            if (gl.getExtension('EXT_color_buffer_float') && gl.getExtension('OES_texture_float_linear')) {
```

Comment:
This is a negative test.  It's better if you add the new test case to a new float texture test with extensions.